sceptre/scipool/templates/bmgfki/lambda-budgets/budget-rules.yaml

```diff
@@ -0,0 +1,14 @@
+# budget rules per user by synapse team id
+teams:
```

Comment:
We need new teams for BMGF-Ki, as well as a new notification email endpoint.

Reply:
done, new team is `3432633`  BMGFKI_ServiceCatalogUsers